calcom/cal.com
# fix: Legacy Routing Forms handling - Allow team Event in existing non-team Routing forms

## What does this PR do?

Fixes #9853

Allow an existing User Routing Form to link to a Team EventType if it's already selected.

## Type of change

- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?

- Locally [comment out the code that](https://github.com/calcom/cal.com/pull/9852/files#diff-243a8c40cc62828c5dc7d4683aee4d4541c773c7556937af97f204838d5a82ddR115) filters out eventTypes
- Add a team EventType to a User Routing Form
- Revert the change and then see that the selection is still there
- Now select a User Event Type and save. Refresh and see that the team event type that was selected earlier has vanished from the list.

## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
